### ISSUE TITLE
Example to use alternative Loss Function

### ISSUE THREAD

**Issue by dhbrand** [hyperopt/hyperopt-sklearn#125]
Do you have an example of how to use a different loss function in the estimator?  I want to use recall or f1 score from sklearn but they require the predicted y.  Not sure how I would program that before the model has been chosen.

**Comment by driesvr** [comment]
There's an example in the pull request below:
https://github.com/hyperopt/hyperopt-sklearn/pull/47
